bench/cli.mjs

```
@@ -0,0 +1,152 @@
+  filtered = filtered.filter((s) => s.operation === operationFilter);
+}
+
+if (filtered.length === 0) {
+  console.error("No matching benchmarks found for the given filters.");
+  showHelp();
+  process.exit(1);
+}
+
+// ── Run benchmarks ─────────────────────────────────────────────────────────
+
+console.log(`\nRunning ${filtered.length} benchmark(s) with size=${size}...\n`);
```

ESLint

[error] 'console' is not defined.

(no-undef)